Changes

```diff
@@ -204,6 +204,10 @@ Working version
   to ill-typed columns
   (Thomas Refis, with help from Gabriel Scherer and Luc Maranget)
 
+- MPR#7701,GPR#1561: do not drop documentation comments in (otherwise) empty
```

Comment:
+ MPR#7138

Reply:
I missed that 7701 was a duplicate. Fixed.